src/pages/home/index.tsx

```diff
@@ -0,0 +1,35 @@
+import Head from "next/head"
+import HeroSection from "./components/HeroSection/HeroSection"
+import HowItWorksSection from "./components/HowItWorksSection/HowItWorksSection"
+
+const Home = () => {
+  return (
+    <>
+      <Head>
+        <title>Home | Festwrap</title>
+        <meta
+          name="description"
+          content="Spotify playlists Generator using festival line-up"
+        />
+        <meta
+          name="keywords"
+          content="spotify, generator, playlist, festival, line-up"
+        />
+        <meta property="og:title" content="Home" />
+        <meta
+          property="og:description"
+          content="Descripción para redes sociales."
```

Comment:
This description and the url, image and twitter ones seem to have placeholder values. Should we update them?

Reply:
I have kept this metadata to update in the next iterations, but I can remove it and add it if we need to in the future.

Reply:
I'd rather remove it and add back when needed.